redash/handlers/alerts.py

```diff
@@ -46,6 +46,33 @@ def delete(self, alert_id):
         models.db.session.delete(alert)
         models.db.session.commit()
 
+class AlertMuteResource(BaseResource):
+    def post(self, alert_id):
+        alert = get_object_or_404(models.Alert.get_by_id_and_org, alert_id, self.current_org)
+        require_admin_or_owner(alert.user.id)
+
+        setattr(alert.options, 'muted', True)
+        models.db.session.commit()
```

Comment:
@arikfr @rauchy I'm attempting to update `alert.options.muted` but this doesn't save to db.
What's the right way to do this?

Reply:
Have you tried just set property - like this: `alert.options.muted = True` ?

Reply:
IT DOES  🎉 🙏

Reply:
Python is not like JS, `dict.key = sth` != `dict['key'] = sth` :smile:

Reply:
Python is not like JS, `dict.key = sth` != `dict['key'] = sth` :smile:

Reply:
So this would have worked? 
```py
setattr(alert["options"], 'muted', True)
```

Reply:
Nope. `setattr(obj, key, value)` is the same as `obj.key = value`